css/src/components/form/select.scss

```diff
@@ -0,0 +1,113 @@
+/* stylelint-disable selector-no-qualifying-type */
+
+$select-color: $text !default;
+$select-background-color: $body-background !default;
+$select-border-color: $text !default;
+$select-placeholder-color: $text-subtle !default;
+
+$select-hover-color: $text !default;
+$select-hover-border-color: $primary !default;
+$select-focus-border-color: $primary !default;
+
+$select-disabled-color: $text-subtle !default;
+$select-disabled-background-color: $body-background-medium !default;
+$select-disabled-border-color: $table-border-dark !default;
+
+$select-danger-border-color: $danger !default;
+$select-success-border-color: $success !default;
+
+$select-focus-box-shadow-size: 0 0 0 0.125rem !default;
+
+$select-arrow-size: 0.525em !default;
+$select-arrow-border-width: 1px !default;
+$select-arrow-real-width: 0.75em !default;
+
+.select {
+	position: relative;
+
+	select {
+		@include control;
+
+		width: 100%;
+		max-width: 100%;
+		border-color: $select-border-color;
+		background-color: $select-background-color;
+		color: $select-color;
+		font-size: inherit;
+		cursor: pointer;
+
+		&[multiple] {
+			padding: 0;
+			overflow-y: auto;
+
+			option {
+				padding: $control-padding-vertical $control-padding-horizontal;
+			}
+		}
+
+		&[disabled] {
+			border-color: $select-disabled-border-color;
+			background-color: $select-disabled-background-color;
+			color: $select-disabled-color;
+		}
+
+		@include focus-visible {
+			border-color: $select-focus-border-color;
+			outline: none !important;
```

Comment:
To ensure forced colors mode gets the same thick outline, lets use:
```
outline-width: 0.125rem; // variable
outline-style: solid;
outline-color: transparent; // forced=colors will override this with the user's preferred focus outline color.
outline-offset: 0;
```

Same change will be needed in other controls.

Reply:
![image](https://user-images.githubusercontent.com/1306747/153647683-91dde874-6e46-4490-9e9a-1bbc3e614ded.png)